lib/convenient_service/configs/standard.rb

```diff
@@ -64,18 +66,24 @@ module Standard
           use Plugins::Common::NormalizesEnv::Middleware
 
           use Plugins::Service::HasResultShortSyntax::Success::Middleware
+
+          use Plugins::Service::DisplaysResult::Success::Middleware
```

Comment:
Please, make your intention more explicit. For example: `DisplaysResultForDebugging`.

Thanks.